src/GuardClauses.UnitTests/GuardAgainsOutOfRange.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Ardalis.GuardClauses;
+using System;
+using Xunit;
+
+namespace GuardClauses.UnitTests
+{
+    public class GuardAgainsOutOfRange
+    {
+        [Fact]
+        public void DoesNothingGivenInRangeValueUsingShortcutMethod()
+        {
+            Guard.AgainsOutOfRange(2, "index", 1, 5);
+        }
+
+        [Fact]
+        public void DoesNothingGivenInRangeValueUsingExtensionMethod()
+        {
+            Guard.Against.OutOfRange(2, "index", 1, 5);
```

Comment:
Same as above. Test at least 1 and 5 in addition to 2. Test all of 1 through 5 if you like, though that's probably not necessary.